lib/validateCreateArgs.js

```diff
@@ -177,19 +178,21 @@ function validateCreateArgs(args) {
         if (args.meta && args.meta['wt-compiler'] && args.meta['wt-compiler'] !== MIDDLWARE_COMPILER) {
             throw Cli.error.invalid('Use of middleware is incompatible with a custom webtask compiler.');
         }
-
+        const filter = mw => mw.startsWith('http') ? 'urls' : 'modules';
```

Comment:
Since we support `node@>=4.2.x`, we can't rely on `String.prototype.startsWith` as it is only supported as of `node@6.12.0` (see: http://node.green/#ES2015-built-ins-well-known-symbols-Symbol-match--String-prototype-startsWith)